src/fido/types.h

```diff
@@ -202,6 +204,7 @@ typedef struct fido_cbor_info {
 	uint64_t          maxcredcntlst; /* max number of credentials in list */
 	uint64_t          maxcredidlen;  /* max credential ID length */
 	uint64_t          fwversion;     /* firmware version */
+	uint64_t	  maxcredbloblen; /* maximum credBlob length */
```

Comment:
s/maximum/max/ for consistency w/ other comments on the same struct?

Reply:
fixed in 3e7e37e1ead79f0539b18bafee58039cc618c3ea